data/successors/successor fleets.txt

```diff
@@ -163,6 +163,7 @@ fleet "People's Houses Raiders"
 fleet "Small Old Houses"
 	personality
 		heroic
+		confusion 15
```

Comment:
According to [the wiki](https://github.com/endless-sky/endless-sky/wiki/ShipPersonalities#special-flags), `confusion` has a default of 10 if it is unset, and 10 is also the standard for "military ships," which I think all of the High Houses fleets qualify as, so I don't think it's necessary to change for any of these unless you have a specific reason for the higher-than average confusion.

```suggestion
```

Reply:
True, I had not remembered it defaulted.

Reply:
I think 10 is fine. It's a pretty minor stat regardless and keeping it at the default seems sensible given I don't think there's any pressing reason it should be otherwise.

Reply:
They fight less often... but that doesn't necessarily mean they're worse.